ci/before_install.sh

```diff
@@ -11,3 +11,7 @@ echo "inside $0"
 sudo apt-get update $APT_ARGS # run apt-get update for all versions
 
 true # never fail because bad things happened here
+
+if [ x"$DOC_BUILD" == x"" ]; then
+    exit 1
+fi
```

Comment:
you nee to put this before the 'true' above. IOW, the script should *always* return true.
(I think this was for testing only?)